output/create_kubevirt_yaml.ml

```diff
@@ -240,3 +251,49 @@ let create_kubevirt_yaml source inspect
 
   (* Return the YAML document. *)
   Doc (Assoc body)
+
+and kubevirt_instancetype guestcaps inspect =
+  (* This only matters for Windows guests below, in which case virtio-win
+   * has either installed all the virtio drivers or none, so we only
+   * need to test one field of guestcaps.
+   *)
```

Comment:
I'll align the names in the next release, tracked by https://github.com/kubevirt/common-instancetypes/issues/188.